src/app/components/authDialog/authDialog.component.ts

```diff
@@ -1,23 +1,16 @@
 import { Component, Inject } from '@angular/core';
 import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material';
-import { AuthService } from '../../services/api/auth.service';
+import { AuthService } from '../../services/auth.service';
 
 @Component({
   selector: 'rm-auth-dialog',
   templateUrl: './authDialog.component.html',
   styleUrls: ['./authDialog.component.scss'],
 })
 export class AuthDialogComponent {
-  username = '';
-  password = '';
+  constructor(private dialogRef: MatDialogRef<AuthDialogComponent>, @Inject(MAT_DIALOG_DATA) public data: any) { }
```

Comment:
(just a question) Why do you need  `@Inject(MAT_DIALOG_DATA) public data: any`

Reply:
Technical requirement from material2